### PR TITLE
S2task1 add bike form

### PR DESCRIPTION
**Pull request by RichVelo**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
+html {
```
Comment (cfbuckley14) [inline style.css:1]: this css should go in the <body> instead of <html>

File: style.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    display: flex;
+    flex-flow: column wrap;
+    border-top: solid 2px #888888;
+}
+
+.newBike h2 {
+    font-size: 2.75rem;
+}
+
+.newBikeForm form {
```
Comment (cfbuckley14) [inline style.css:58]: see comment regarding unnecessary class

File: style.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+.newBike {
+    margin: 0 auto;
+    width: 60%;
+    height: 50vh;
+    display: flex;
+    flex-flow: column wrap;
+    border-top: solid 2px #888888;
+}
+
+.newBike h2 {
```
Comment (cfbuckley14) [inline style.css:54]: you only have one <h2> so don't require the class here to target

File: style.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    gap: 10px 10px;
+    padding: 10px;
+    justify-content: space-evenly;
+}
+
+@media screen and (max-width: 650px) {
+    .bikeContainer {
+        flex-direction: column;
+    }
+    .bikeCard{
```
Comment (cfbuckley14) [inline style.css:72]: space after .bikeCard

Comment (RichVelo) [reply style.css:72]: 👍

File: style.css (hunk 2 above)
Comment (RichVelo) [reply style.css:58]: 👍

File: index.php [hunk 5]
```diff
@@ -1,19 +1,57 @@
...
+        <h1>Richie's excellent collection of bikes</h1>
+    </header>
+    <main>
+        <section class="bikeContainer">
+            <?= bikeDetails($allBikes)?>
+        </section>
+        <section class="newBike">
+            <h2>N+1</h2>
+            <h3>Add more bikes here</h3>
+            <div class="newBikeForm">
```
[the remark this answers is not in the document]
Comment (RichVelo) [reply index.php:40]: 👍

**Pull request merged**